vignettes/non-english.Rmd

```diff
@@ -0,0 +1,79 @@
+---
+title: "Non-English sites"
+output: rmarkdown::html_vignette
+description: > 
+  If your documentation is written in a language other than English, you can 
+  generate text in your language using the `lang` field. You will also learn
+  how to provide your own translations if your language isn't currently 
+  supported by pkgdown.
+vignette: >
+  %\VignetteIndexEntry{Non-English sites}
+  %\VignetteEngine{knitr::rmarkdown}
+  %\VignetteEncoding{UTF-8}
+---
+
+```{r, include = FALSE}
+knitr::opts_chunk$set(
+  collapse = TRUE,
+  comment = "#>"
+)
+```
+
+If your documentation (`.Rd` and `.Rmd`) is written in a language other than English, declare it by setting setting `lang` to the [two letter language code](https://en.wikipedia.org/wiki/List_of_ISO_639-1_codes) for your language:
+
+``` yaml
+lang: fr
+```
+
+This will be used to set the language of the web page and to translate the English words that pkgdown generates on your site.
+Current available translations are:
+
+-   `ca`: Catalan
+-   `de`: German
+-   `dk`: Danish
+-   `es`: Spanish
+-   `fr`: French
+-   `ko`: Korean
+-   `pt`: Portuguese
+-   `tr`: Turkish
+-   `zh_CN`: Chinese (simplified)
+
+## Translations
+
+Translations are contributed by community members so if your language is not currently available, you could be the one to add it! 
```

Comment:
should people first open an issue indicating they're working on a translation? so any contributor would first check whether there's such an issue, and potentially volunteer as a reviewer for that language?

Reply:
Yeah, that's worth suggesting.